### PR TITLE
Require Jenkins 2.361 LTS and Java 11

### PR DESCRIPTION
**Pull request by offa**
Require Jenkins 2.361 LTS and Java 11.

:warning: The Jenkinsfile change needs someone with write access to take effect.

----------------------------------------

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
...
-  [ platform: "windows", jdk: "8" ],
-  [ platform: "linux", jdk: "11" ],
-  [ platform: "linux", jdk: "17", jenkins: "2.342" ],
-])
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 11],
+    [platform: 'windows', jdk: 11]
+])
```
Comment (lemeurherve) [inline Jenkinsfile:6]:
```suggestion
])

```
Missing EOL (nit)

**Comment by lemeurherve** [comment]
@offa I've replayed this build with the updated Jenkinsfile.

Don't hesitate to ping me on the similar PRs you're opening (thanks btw!) so I can do the same on them.

**Comment by offa** [comment]
Thanks! :-)

**Comment by offa** [comment]
Conflicts resolved.

**Comment by dwnusbaum** [comment]
Superseded by https://github.com/jenkinsci/script-security-plugin/pull/488.

**Pull request closed without merging**